src/read.c

```diff
@@ -832,16 +832,14 @@ typedef struct avifMeta
     // Contents of grpl box, which signal groups of entities (items or tracks).
     avifEntityToGroups entityToGroups;
 
-#if defined(AVIF_ENABLE_EXPERIMENTAL_MINI)
-    // If true, the fields above were extracted from a MinimizedImageBox.
-    avifBool fromMiniBox;
-#endif
-
-#if defined(AVIF_ENABLE_EXPERIMENTAL_SAMPLE_TRANSFORM)
     // Parsed from Sample Transform metadata if present, otherwise empty.
     avifSampleTransformExpression sampleTransformExpression;
     // Bit depth extracted from the pixi property of the Sample Transform derived image item, if any.
     uint32_t sampleTransformDepth;
+
+#if defined(AVIF_ENABLE_EXPERIMENTAL_MINI)
+    // If true, the fields above were extracted from a MinimizedImageBox.
```

Comment:
IMPORTANT:  Please check if this comment is still correct after it is moved after the two sample transform fields. If not, we need to move the `fromMiniBox` field back to its original location.